.github/workflows/ci.yaml

```diff
@@ -210,7 +210,7 @@ jobs:
             SANDBOX_ID="default"
             CHART_REF="$GITHUB_SHA"
             KUBERNETES_ENV="production"
-            REPLICA_COUNT="4"
+            REPLICA_COUNT="1"
```

Comment:
if we need to reduce this, I would suggest at least 3, running the pipeline needs 2 pods, so it'd be great to have a third one if another user is working too (I would keep it to 4)